src/meson.build

```diff
@@ -54,12 +53,12 @@ executable(
     asresources,
     config_header,
     dependencies: [
-        dependency('gtk+-3.0'),
-        dependency('granite', version: '>= 5.2.0'),
-        dependency('gee-0.8'),
-        dependency('libxml-2.0'),
-        dependency('cairo', version: '>=1.14'),
-        dependency('goocanvas-2.0'),
+        gtk_dependency,
```

Comment:
What's the benefit of this?
I'm just curious, I have nothing against this approach, but wouldn't this mean that everything we need to update a dependency, or add a new one, we need to do it twice in 2 different files?

Reply:
Well, this allows the packagers to just check the main `meson.build` file to get an idea of which libraries we depend on.